interpreter/cling/include/cling/Interpreter/RuntimePrintValue.h

```diff
@@ -232,6 +235,13 @@ namespace cling {
       }
       return str + " }";
     }
+
+#if __cplusplus >= 201703L
+    // For std::filesystem::path
+    inline std::string printValue_impl(const std::filesystem::path* obj) {
```

Comment:
Why do we have to go through `printValue_impl`, why isn't `printValue` enough here (as we do for `std::string`, `std::source_location` and others)?

Reply:
My bad, it should've been `printValue`. Done!